go.mod

```diff
@@ -5,5 +5,10 @@ go 1.12
 require (
 	github.com/ciscoecosystem/aci-go-client v1.35.2
 	github.com/ghodss/yaml v1.0.0
+	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320 // indirect
+	github.com/hashicorp/hcl/v2 v2.3.0 // indirect
+	github.com/hashicorp/terraform-json v0.13.0
+	github.com/hashicorp/terraform-plugin-go v0.2.1 // indirect
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.3
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
```

Comment:
Seems that those updates could be achieve also by bumping the github.com/hashicorp/terraform-plugin-sdk/v2 to a more recent version. Any reason not to bump the sdk version directly?